script.colorbox/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.colorbox" name="colorbox Script" version="2.0.3" provider-name="BADMS">
+<addon id="script.colorbox" name="colorbox Script" version="2.0.4" provider-name="BADMS">
```

Comment:
You changed the license type so please also update the `license` tag in the addon.xml